doc/tutorials/space_shooter/step_4.md

```diff
@@ -48,7 +47,10 @@ class Bullet extends SpriteAnimationComponent
     with HasGameReference<SpaceShooterGame> {
   Bullet({
     super.position,
-  }) : super(size: Vector2(25, 50));
+  }) : super(
+          size: Vector2(25, 50),
+          anchor: Anchor.center,
+        );
```

Comment:
Seems like the formatting of all these are wrong?

Reply:
not really, this is the result of `dart format` command, I agree that it looks weird hahaha